src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -6,7 +6,23 @@
     public static final String OAUTH_AUTHENTICATION_SCHEME = "OAuth";
 
     public static final int VALIDATION_RESULT_MESSAGE_MAX_LENGTH = 2000;
-
+    
+    public static class AccountingStringValidationErrorMessages {
+        public static final String ERROR_CHART_OF_ACCTS_REQUIRED = "Chart is a required field.";
```

Comment:
Thank you for pointing this out. I was wondering if it would make more sense to use the KFS error messages in the properties files or use constants. I used constants because initially I thought that if we move this into a micro-service that calls KFS endpoints to check on whether accounts, object codes, etc exist it would be better not to depend on the KFS error messages in the properties files. After further discussions about possible micro-services I think we talked that perhaps the validation service could be an end point in KFS in which case using the KFS error messages in the properties files would make sense. What do you think?